app/src/main/java/org/sopt/androidassignment1/SignInActivity.kt

```diff
@@ -0,0 +1,40 @@
+package org.sopt.androidassignment1
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import org.sopt.androidassignment1.databinding.ActivitySigninBinding
+
+class SignInActivity : AppCompatActivity() {
+    private lateinit var binding: ActivitySigninBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivitySigninBinding.inflate(layoutInflater)
+
+        setContentView(binding.root)
+
+        var id = binding.idInput.text
+        var pw = binding.pwInput.text
```

Comment:
```suggestion
        val id = binding.idInput.text
        val pw = binding.pwInput.text
```
id와 pw 변수의 값이 향후 변하지 않는다면 val로 선언하시는 게 좀 더 좋을 것 같아요 :)